client/src/pages/patients/register/PatientRegistrationAccordion.jsx

```diff
@@ -111,11 +111,8 @@ export default function PatientRegistrationAccordion ({
               <TextInput
                 type='date'
                 label='Date of Birth'
-                valueFormat='YYYY-MM-DD'
-                placeholder='YYYY-MM-DD'
-                defaultLevel='decade'
-                maxDate={new Date()}
                 withAsterisk
+                max={new Date().toLocaleDateString('en-CA')} // need to use en-CA to get the correct format for HTML date input
```

Comment:
Is en-CA date format guaranteed to be consistent?

I see in our client package.json file that we have the `dayjs` package where we could use a literal date format (i.e. YYYY-MM-DD).

Reply:
Good point! Switched to using `dayjs` which was installed when we were using the Mantine dates package. Replaced other uses of `toLocaleDateString` with `dayjs` too